blackbox/docs/protocols/postgres.txt

```diff
@@ -34,21 +32,18 @@ Therefore clients should generally enable ``autocommit``.
 Server Compatibility & Implementation Status
 ============================================
 
-Crate emulates server version ``9.5``
+Crate emulates a PostgreSQL server version ``9.5``
 
 Start-up
 --------
 
 The connection handshake is implemented without authentication support. Crate
-will always respond with ``AuthenticationOK``.
-If a Client sends a SSLRequest the server will answer with ``N`` to indicate
-that it doesn't support SSL.
-
-The ``database`` name a client sends will be used as default schema as Crate
-doesn't have database support.
+will always respond with ``AuthenticationOK`` and any SSLRequests are answered
+with ``N`` to indicate that it does not support SSL.
 
-The server will inform the Client that ``UTF8`` encoding should be used. Any
-other encoding is not supported.
+Since Crate uses schemas instead of databases, the ``database`` parameter sets
+the default schema name for future queries. Usually this schema is ``doc``.
```

Comment:
I wouldn't say usually but: Default schema if no database parameter is specified